tools/send-to-caas/send-utils.js

```diff
@@ -30,27 +36,34 @@ const LOCALES = {
   dk: { ietf: 'da-DK' },
   de: { ietf: 'de-DE' },
   ee: { ietf: 'et-EE' },
+  eg_ar: { ietf: 'ar-EG' },
+  eg_en: { ietf: 'en-GB' },
   es: { ietf: 'es-ES' },
   fr: { ietf: 'fr-FR' },
   gr_en: { ietf: 'en-GR' },
+  gr_el: { ietf: 'el-GR' },
   ie: { ietf: 'en-IE' },
   il_en: { ietf: 'en-IL' },
+  il_he: { ietf: 'he-il' },
   it: { ietf: 'it-IT' },
+  kw_ar: { ietf: 'ar-KW' },
+  kw_en: { ietf: 'en-GB' },
   lv: { ietf: 'lv-LV' },
   lt: { ietf: 'lt-LT' },
   lu_de: { ietf: 'de-LU' },
   lu_en: { ietf: 'en-LU' },
   lu_fr: { ietf: 'fr-LU' },
   hu: { ietf: 'hu-HU' },
   mt: { ietf: 'en-MT' },
-  mena: { ietf: 'en-mena' },
   mena_en: { ietf: 'en-mena' },
   mena_ar: { ietf: 'ar-mena' },
-  mena_fr: { ietf: 'fr-mena' },
```

Comment:
Deleted as `mena_fr` isn't supported when rolled out and to keep parity with the 93 values supported by Milo's `getLocale()` api